install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -115,6 +115,7 @@ type ServerConfig struct {
 	OAuthServer                       OAuthServer       `json:"oauthServer"`
 	Session                           Session           `json:"session"`
 	GithubApp                         *GithubApp        `json:"githubApp"`
+	ChargebeeSecret                   string            `json:"chargebeeSecret"`
```

Comment:
I'd like to use an [ObjectRef](https://github.com/gitpod-io/gitpod/blob/main/install/installer/pkg/config/v1/config.go#L176) here rather than a `string`, but that would require creating an import cycle `config -> experimental -> config`

Reply:
Is it worth to refactor this? Maybe something for a follow-up issue?

Reply:
I think it's not worth following up on at this stage.